electricpy/__init__.py

```diff
@@ -1623,42 +1660,75 @@ def dynetz(delta=None,wye=None,round=None):
                 Wye-Connected impedance values { Z1, Z2, Z3 }
     """
     # Determine which set of impedances was provided
-    if(delta!=None and wye==None):
-        Z12, Z23, Z31 = delta # Gather particular impedances
-        Zsum = Z12 + Z23 + Z31 # Find Sum
+    if (delta != None and wye == None):
+        Z12, Z23, Z31 = delta  # Gather particular impedances
+        Zsum = Z12 + Z23 + Z31  # Find Sum
         # Calculate Wye Impedances
-        Z1 = Z12*Z31 / Zsum
-        Z2 = Z12*Z23 / Zsum
-        Z3 = Z23*Z31 / Zsum
-        Zset = ( Z1, Z2, Z3 )
-        if round!=None: Zset = _np.around(Zset,round)
-        return(Zset) # Return Wye Impedances
-    elif(delta==None and wye!=None):
-        Z1, Z2, Z3 = wye # Gather particular impedances
-        Zmultsum = Z1*Z2 + Z2*Z3 + Z3*Z1
+        Z1 = Z12 * Z31 / Zsum
+        Z2 = Z12 * Z23 / Zsum
+        Z3 = Z23 * Z31 / Zsum
+        Zset = (Z1, Z2, Z3)
+        if round != None: Zset = _np.around(Zset, round)
+        return (Zset)  # Return Wye Impedances
+    elif (delta == None and wye != None):
+        Z1, Z2, Z3 = wye  # Gather particular impedances
+        Zmultsum = Z1 * Z2 + Z2 * Z3 + Z3 * Z1
         Z23 = Zmultsum / Z1
         Z31 = Zmultsum / Z2
         Z12 = Zmultsum / Z3
-        Zset = ( Z12, Z23, Z31 )
-        if round!=None: Zset = _np.around(Zset,round)
-        return(Zset) # Return Delta Impedances
+        Zset = (Z12, Z23, Z31)
+        if round != None: Zset = _np.around(Zset, round)
+        return (Zset)  # Return Delta Impedances
     else:
         raise ValueError("ERROR: Either delta or wye impedances must be specified.")
 
-# Define Single Line Power Flow Calculator
-def powerflow( Vsend, Vrec, Zline ):
+#calculating impedance of bridge network
+def bridge_impedance(z1, z2, z3, z4, z5):
     """
-    Simple Power-Flow Calculator
+    bridge impedance calculator
     
-    This function is designed to calculate the ammount of real
-    power transferred from the sending end to the recieving end
-    of an electrical line given the sending voltage (complex),
-    the receiving voltage (complex) and the line impedance.
+    z1*z3 == z2*z4 this is a condition of wheat stone bridje so current through z5 will be zero
     
+    Parameters
+    ----------
+    z1, z2, z3, z4 are bridge impedance's in clock wise direction
```

Comment:
Is the circuit you're describing here similar to this:

![image](https://user-images.githubusercontent.com/33275230/103730692-48cdc400-4f98-11eb-8b5c-7cc8d11e4e44.png)

Seems like what you're describing doesn't exactly match that image, but I want to make sure I fully understand it, since we'll want to add an actual image of the circuit for simplicity.

Reply:
![20210107_091257 (2)](https://user-images.githubusercontent.com/52835045/103848648-ddcfda80-50c8-11eb-9934-5c67f46c0169.jpg)
Sir this what I thought about wheatstone stone bridge

Reply:
This conversation has continued in the subsequent PR. 
https://github.com/engineerjoe440/ElectricPy/pull/6#issuecomment-760618669